src/main/java/io/vavr/control/Validation.java

```diff
@@ -258,6 +325,16 @@ private Validation() {
     /**
      * Combines five {@code Validation}s into a {@link Builder5}.
      *
+     *  <pre>{@code
+     *  //following code joins five validations into one which result in Validation.Builder5
+     * Validation validation1 = Validation.valid("vavr1")
+     * Validation validation2 = Validation.valid("vavr2")
+     * Validation validation3 = Validation.valid("vavr3")
+     * Validation validation4 = Validation.valid("vavr4")
+     * Validation validation5 = Validation.valid("vavr5")
+     * validation1.combine(validation2,validation3,validation4,validation5)
```

Comment:
Please generally add a space after all occurrences of `,` for reasons of readability.
Please look at our source code, we do it everywhere.